README.md

```diff
@@ -1,135 +1,57 @@
-# Auction system
+[<img src="docs/logo.png" alt="Lagom" width="50%">](https://github.com/lagom/lagom)
 
-System architecture:
+[![Gitter](https://img.shields.io/gitter/room/gitterHQ/gitter.svg)](https://gitter.im/lagom/lagom?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [<img src="https://img.shields.io/travis/lagom/lagom.svg"/>](https://travis-ci.org/lagom/lagom)
```

Comment:
Now that we have a Travis CI build for this project, it probably makes sense to use the badge for that build rather than the main `lagom/lagom` one https://travis-ci.org/lagom/online-auction-scala